src/System.ComponentModel.Annotations/tests/EditableAttributeTests.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Xunit;
+
+namespace System.ComponentModel.DataAnnotations
+{
+    public class EditableAttributeTests
+    {
+        [Fact]
+        public static void Can_construct_and_both_AllowEdit_and_AllowInitialValue_are_set()
+        {
+            var attribute = new EditableAttribute(true);
+            Assert.Equal(true, attribute.AllowEdit);
+            Assert.Equal(true, attribute.AllowInitialValue);
```

Comment:
What about:
- When the value passed to EditableAttribute's ctor is false?
- When AllowInitialValue is changed, making sure AllowEdit isn't also changed?